config/initializers/training_routes.rb

```diff
@@ -245,10 +232,173 @@
+      AllocationSubjects::ANCIENT_LANGUAGES,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:school_direct_tuition_fee],
+    amount: 10_000,
+    allocation_subjects: [
+      AllocationSubjects::BIOLOGY,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:opt_in_undergrad],
+    amount: 9_000,
+    allocation_subjects: [
+      AllocationSubjects::MATHEMATICS,
+      AllocationSubjects::PHYSICS,
+      AllocationSubjects::COMPUTING,
+      AllocationSubjects::MODERN_LANGUAGES,
+    ],
+  ),
+].freeze
+
+SCHOLARSHIPS_2023 = [
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:provider_led_postgrad],
+    amount: 26_000,
+    allocation_subjects: [
+      AllocationSubjects::CHEMISTRY,
+      AllocationSubjects::COMPUTING,
+      AllocationSubjects::MATHEMATICS,
+      AllocationSubjects::PHYSICS,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:school_direct_tuition_fee],
+    amount: 26_000,
+    allocation_subjects: [
+      AllocationSubjects::CHEMISTRY,
+      AllocationSubjects::COMPUTING,
+      AllocationSubjects::MATHEMATICS,
+      AllocationSubjects::PHYSICS,
+    ],
+  ),
+].freeze
+
+GRANTS_2023 = [
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:early_years_salaried],
+    amount: 14_000,
+    allocation_subjects: [
+      AllocationSubjects::EARLY_YEARS_ITT,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:school_direct_salaried],
+    amount: 24_000,
+    allocation_subjects: [
+      AllocationSubjects::CHEMISTRY,
+      AllocationSubjects::COMPUTING,
+      AllocationSubjects::MATHEMATICS,
+      AllocationSubjects::PHYSICS,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:school_direct_salaried],
+    amount: 15_000,
+    allocation_subjects: [
+      AllocationSubjects::DESIGN_AND_TECHNOLOGY,
+      AllocationSubjects::GEOGRAPHY,
+      AllocationSubjects::MODERN_LANGUAGES,
+      AllocationSubjects::ANCIENT_LANGUAGES,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:school_direct_salaried],
+    amount: 10_000,
+    allocation_subjects: [
+      AllocationSubjects::BIOLOGY,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:pg_teaching_apprenticeship],
+    amount: 15_000,
+    allocation_subjects: [
+      AllocationSubjects::CHEMISTRY,
+      AllocationSubjects::COMPUTING,
+      AllocationSubjects::MATHEMATICS,
+      AllocationSubjects::PHYSICS,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:pg_teaching_apprenticeship],
+    amount: 6_000,
+    allocation_subjects: [
+      AllocationSubjects::DESIGN_AND_TECHNOLOGY,
+      AllocationSubjects::GEOGRAPHY,
+      AllocationSubjects::MODERN_LANGUAGES,
+      AllocationSubjects::ANCIENT_LANGUAGES,
+    ],
+  ),
+  OpenStruct.new(
+    training_route: TRAINING_ROUTE_ENUMS[:pg_teaching_apprenticeship],
+    amount: 6_000,
```

Comment:
```suggestion
    amount: 1_000,
```

Going by our funding spreadsheet and [the funding manual for teaching apprenticeships](https://www.gov.uk/government/publications/postgraduate-teaching-apprenticeships-funding-manual/postgraduate-teaching-apprenticeship-funding-manual-2022-to-2023-academic-year#grant-funding-rates), this is £1000.